pymeasure/instruments/keithley/keithley2602B.py

```diff
@@ -50,21 +55,45 @@ class Keithley2602B(Keithley2600):
             f"smu{channel}.trigger.IDLE_EVENT_ID",
         ]
 
-    def __init__(self, adapter, **kwargs):
-        super().__init__(adapter, includeSCPI=False, **kwargs)
+    def __init__(self, hostname: str, port: int = 5025, **kwargs) -> None:
+        """Initializes LAN communication with the device.
+        Flushes the error queue and logs the system identification info.
+        If ID info cannot be retrieved, or communication was not successfully
+        initialized, the results are logged.
 
+        Args:
+            hostname (str): Hostname of the system (initial expected value is a concatenation of
+                product number and serial number. Note that the hostname can be overwritten.
+            port (int): Port used to communicate with the system
+        """
+        self.hostname = hostname
+        self.port = port
+        adapter = f"TCPIP::{self.hostname}::{self.port}::SOCKET"
         self.dio_pins = [
             Keithley2600DigitalIOPin(self, i + 1) for i in range(self.number_of_pins)
         ]
 
+        try:
+            super().__init__(adapter, **kwargs)
+
+        except ValueError or socket_error:
+            log.exception(f"Could not connect to {self.hostname}", exc_info=True)
```

Comment:
```suggestion
            log.exception(f"Could not connect to {self.hostname}")
```
Redundant, as written. My understanding is that, within an exception handler, `log.exception("some text")` is equivalent to `log.error("some text", exc_info=True)`.